x-pack/plugins/security_solution/public/detection_engine/rule_details_ui/pages/rule_details/execution_log_table/execution_log_table.tsx

```diff
@@ -432,11 +454,21 @@ const ExecutionLogTableComponent: React.FC<ExecutionLogTableProps> = ({
 
   const executionLogColumns = useMemo(() => {
     const columns = [...EXECUTION_LOG_COLUMNS];
+    let messageColumnWidth = 50;
+
+    if (showSourceEventTimeRange) {
+      columns.push(...getSourceEventTimeRangeColumns());
+      messageColumnWidth = 30;
+    }
```

Comment:
We need to check that the feature flag is enabled here as well.

Reply:
We still need to cover the situation when the feature flag gets enabled and then disabled for any reason. When I disabled the feature flag locally, the table continued showing this column.